src/main/java/uk/co/fivium/dmda/emailmessages/EmailMessage.java

```diff
@@ -127,7 +127,12 @@ private void readMessageBody(){
 
       while(lHeaderEnumeration.hasMoreElements()){
         Header lCurrentHeader = Header.class.cast(lHeaderEnumeration.nextElement());
-        mHeaderMap.put(lCurrentHeader.getName(), decodeHeaderValue(lCurrentHeader));
+        try {
+          mHeaderMap.put(lCurrentHeader.getName(), decodeHeaderValue(lCurrentHeader));
+        }
+        catch(MessagingException ex) {
+          mLogger.warn("Failed to decode header value for (" + lCurrentHeader.getName() + ")" + toString(), ex);
```

Comment:
Nit: I know you were just copying what's already there, but you can use the logger string interpolation with `{}` rather than concatenating the string together which is a bit nicer eg.
`mLogger.warn("Failed to decode value for header {}", lCurrentHeader.getName(), ex);`

Reply:
Thanks. Fixed.